tasks/jsinspect.js

```diff
@@ -24,9 +25,21 @@ module.exports = function(grunt) {
       identifiers: false,
       failOnMatch: true,
       suppress:    100,
-      reporter:    'default'
+      reporter:    'default',
+      configFile:  '.jsinspectrc'
     });
 
+    if (fs.existsSync(options.configFile) && fs.lstatSync(options.configFile).isFile()) {
+      var contents = strip(fs.readFileSync(options.configFile, 'utf8'));
```

Comment:
I'm not agreeing on that one.

npm pushes semantic version and I think we should stick to that.
Meaning, when we break functionality we're on `1.0.0`.

> 0.5.0 or similar it should be and since we are in 0.x mode, that should be "expected".
Recommend by npm there is no `0.x` unstable "deal with it" mode.

Key rule is to bump major when there is a breaking change and npm wants that ( think of the new default package definition `^1.2.3` which installs up to the next major version ).

What do you think?